### PR TITLE
Minor fixes to scripting

### PR DESCRIPTION
**Pull request by jarrpa**
Noticed a few things to clean up, including waiting for service account secrets to populate.

### CODE REVIEW

File: deploy/gk-deploy [hunk 1]
```diff
@@ -162,13 +162,9 @@ while [[ $# -ge 1 ]]; do
     keypos=1
     while [[ $keypos -lt $keylen ]]; do
       case ${key:${keypos}} in
-        o*|-openshift)
-        CLI="oc"
-        if [ "$key" == "--openshift" ]; then keypos=$keylen; fi
-        ;;
```
Comment (obnoxxx) [inline deploy/gk-deploy]: This removal hunk is (arguably) not a syntax cleanup. :-) What's the reasoning? Should it be a separate commit if at all?

Comment (jarrpa) [inline deploy/gk-deploy]: It's a cleanup. I forgot to remove it earlier when I closed off openshift support and will currently break things if used. Hence, fix.
